src/annotator/test/guest-test.coffee

```diff
@@ -50,12 +46,16 @@ describe 'Guest', ->
   sandbox = sinon.sandbox.create()
   CrossFrame = null
   fakeCrossFrame = null
+  highlighter = null
   guestConfig = null
+  htmlAnchoring = null
 
   createGuest = (config={}) ->
     config = Object.assign({}, guestConfig, config)
     element = document.createElement('div')
-    return new Guest(element, config)
+    guest = new Guest(element, config)
+    guest.anchoring = htmlAnchoring
```

Comment:
Why is this here instead of in the mocked imports? This feels hacky to me.

Reply:
This is necessary because `guest.anchoring` is initialized when the module is first evaluated, and so isn't affected by mocking that happens later. That's one limitation of this approach to mocking. See [this node in the mocking plugin's README](https://github.com/robertknight/babel-plugin-mockable-imports#mocking-code-that-runs-when-a-module-is-imported). The same issue applies to `patch` in Python.

An alternative solution would be to use constructor injection. Something conceptually like this:

```js
class Guest(element, config, anchoring = htmlAnchoring) {
   this.anchoring = anchoring
}
```

With this approach, the default initialization expression is evaluated when the `Guest` constructor is called, so mocking would work.

Reply:
I've pushed a change which does this to handle the issue in a cleaner way ^